.editorconfig

```diff
@@ -0,0 +1,74 @@
+# Visual Studio generated .editorconfig file with C++ settings.
+root = true
+
+[*.{c++,cc,cpp,cppm,cxx,h,h++,hh,hpp,hxx,inl,ipp,ixx,tlh,tli}]
+
+indent_style = space
+indent_size = 4
+insert_final_newline = true
+charset = utf-8
+
+# Visual C++ Code Style settings
+
+cpp_generate_documentation_comments = doxygen_triple_slash
+
+# Visual C++ Formatting settings
+
+cpp_indent_braces = false
+cpp_indent_multi_line_relative_to = statement_begin
+cpp_indent_within_parentheses = indent
+cpp_indent_preserve_within_parentheses = true
+cpp_indent_case_contents = true
+cpp_indent_case_labels = false
+cpp_indent_case_contents_when_block = false
+cpp_indent_lambda_braces_when_parameter = true
+cpp_indent_goto_labels = one_left
+cpp_indent_preprocessor = leftmost_column
+cpp_indent_access_specifiers = false
+cpp_indent_namespace_contents = true
+cpp_indent_preserve_comments = false
+cpp_new_line_before_open_brace_namespace = ignore
+cpp_new_line_before_open_brace_type = ignore
+cpp_new_line_before_open_brace_function = ignore
+cpp_new_line_before_open_brace_block = ignore
+cpp_new_line_before_open_brace_lambda = ignore
+cpp_new_line_scope_braces_on_separate_lines = true
+cpp_new_line_close_brace_same_line_empty_type = false
+cpp_new_line_close_brace_same_line_empty_function = false
+cpp_new_line_before_catch = true
+cpp_new_line_before_else = true
+cpp_new_line_before_while_in_do_while = true
```

Comment:
Did you change this from the default one? If so, why? When I auto-generated the file on my machine it is put to:
`cpp_new_line_before_while_in_do_while = false`

Reply:
Yes, I changed this, prefering it this way
![image](https://user-images.githubusercontent.com/4881321/216313440-4b577e3f-52fa-4d25-b0b3-39f36965468f.png)
as opposed to this
![image](https://user-images.githubusercontent.com/4881321/216313508-aaf34fae-7a29-4c9d-83a5-760d07f35753.png)

Do you see a problem here - are you suggesting "just go with the defaults, less to discuss and think about"?